i3-settings-qubes/qubes-i3status

```diff
@@ -9,7 +9,7 @@ json() {
 }
 
 status_net() {
-    local netvms=$(qvm-ls --raw-data --fields FLAGS 2>/dev/null | grep '...N....' | wc -l)
+    local netvms=$(qvm-ls --no-spinner --raw-data --fields NAME,FLAGS 2>/dev/null | grep '...N...' | cut -d '|' -f1 | xargs"
```

Comment:
`xargs`?!
I'd also add `$` at the end of that regexp, to not match "someName" VM.

Reply:
xargs... I did not notice the loop below...fixed
The regex shoudl now also be more granular.